### DISCUSSION TITLE
Hoymiles HMS 800W-2T Wifi live auslesen

### DISCUSSION THREAD

**Discussion by joshia0** [tbnobody/OpenDTU#1957] [Q&A]
Hallo,

gibt es inzwischen eigentlich eine Möglichkeit, den Hoymiles HMS 800W-2T Wifi mit openDTU auszulesen oder wird dran gearbeitet ?
Dieses Cloudzeug gefällt mir nicht so.
Wäre zu schön ;-)
Danke und lg

**Comment by Ghanrim** [comment]
Häng mich mal dran habe auch 800w-2t

**Comment by tuvok81** [comment]
Ich häng mich hier auch mal dran :)

**Comment by checkersky** [comment]
Also auslesen geht hier mit:  https://github.com/ohAnd/dtuGateway/
benötigt wird dann aber ein ESP8266

**Reply by Frankyistda** [reply]
Hallo zusammen!
Seit heute bin ich auch Besitzer eines HMS-800W-2T und vermisse meine kleine openDTU Anzeige der aktuellen Werte. 
Ich habe vor einem Jahr ohne viel Kenntnis einfach die vorkompilierte opendtu-generic.factory.bin runtergeladen und geflashed. Ging problemlos toll!

Als Hardware habe ich ein ESP32-D0WD-V3 Board, ein Funkmodul E01-ML01DP5 V3.2 basierend auf nRF24L01+ und das 1.3“ Display SH1106.

@checkersky: für Deine Lösung brauche ich aber das einfachere ESP8266 Board? Oder geht das auch mit dem ESP32?

Danke für jede Hilfe!

Gruß
Frank

**Comment by checkersky** [comment]
Hallo, geht mittlerweile auch mit dem esp32.

Lade dir einfach den snapshot 1.9.0047 für den esp32 runter.

**Reply by Frankyistda** [reply]
Danke für die schnelle Antwort!

Nicht böse sein, aber wo finde ich das? Unter:  https://github.com/ohAnd/dtuGateway? 

Hast Du vielleicht einen Link für mich armen Sünder?

Danke schon einmal!

Gruß
Frank

**Reply by Frankyistda** [reply]
Hurra, ich habe es gefunden!

https://github.com/ohAnd/dtuGateway/releases/tag/snapshot

Merci!

**Comment by Frankyistda** [comment]
Sorry, aber das bekomme ich nicht hin, @checkersky .

Wenn ich Dein dtuGateway_ESP32_snapshot_1.9.0048_develop.bin flashe, dann ist kein Accesspoint zu sehen, also auch kein hoymilesGW_xxx.

Wenn ich dagegen das alte opendtu-generic.factory.bin wieder aufspiele, dann ist auch ein Accesspoint OpenDTU-xxx wieder zu sehen und zu verbinden. Nur das alte Programm nützt mir ja nichts mehr.

Was mache ich falsch? Alles?

**Reply by checkersky** [reply]
hi, bei mir funktioniert es mit dem esp32.

Aber wenn du Probleme hast ist es besser wenn du die fragen in richtigen repo stellst der Entwickler von dtugateway wird hier nicht mitlesen somit kann er dir auxh nicht helfen.
